### ISSUE TITLE
No failure details

### ISSUE THREAD

**Issue by aleclaws** [lukejpreston/xunit-viewer#2]
Hi, Thanks for build this tool!

I've been using junit-viewer, but can't move over to xunit-viewer because it doesn't display the details from a failure testcase.

Here is an example of the test case that doesn't display
```
    <testcase name="test_name" time="0.155" >
      <failure><![CDATA[0x1 != 0
./failed_file_name.c:467: error: Failure!]]></failure>
    </testcase>
```

Also, would be great to have the `--contracted` option back

**Comment by aleclaws** [comment]
Was able to resolve this with the following diff:

	$ git diff
	diff --git a/parser/index.js b/parser/index.js
	index 3f2bd4b..ea0cc27 100644
	--- a/parser/index.js
	+++ b/parser/index.js
	@@ -82,6 +82,8 @@ const extractTestCore = (test, type, status) => {
	         test.message = ''
	         if (core['$'].message) test.message += core['$'].message
	         if (core['$'].type) test.message += core['$'].type
	+      } else if (typeof core === 'string') {
	+        test.message = core
	       }
	     }

**Comment by lukejpreston** [comment]
create a separate issue for the contracted please, I wasn't sure if anyone actually used it so I decided not to add it

**Comment by lukejpreston** [comment]
this should be fixed in 5.0.2
now you will see below

<img width="948" alt="screenshot 2017-05-18 13 28 38" src="https://cloud.githubusercontent.com/assets/4129128/26202070/eee220b2-3bcd-11e7-9073-11a1ae44376b.png">

the xml parser strips

```
<![CDATA[0x1 != 0
./failed_file_name.c:467: error: Failure!]]>
```

into

```
0x1 != 0 ./failed_file_name.c:467: error: Failure!
```
I don't think i will be changing that any further because that would be an issue with `xml2js`

**Comment by aleclaws** [comment]
Thank you!
